Website/structure-sources/index.rakudoc

```diff
@@ -47,20 +47,20 @@
               <div class="card card-home">
                 <div class="card-content">
                   <div class="has-text-centered">
-                    <a href="/types.html">
+                    <a href="/type">
```

Comment:
This should also be `types`, not `type`, right?

Reply:
this should be types. too much deleted